### PR TITLE
Update mobile-devices-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected invalid link in the doc with https://docs.microsoft.com/en-us/sharepoint/administration/mobile-devices-overview

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/administration/mobile-devices-overview.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ SharePoint Server 2013 offers new, optimized viewing experiences across differen
 > [!IMPORTANT]
 > For more information about supported mobile browsers with SharePoint Server 2013, see [Mobile device browsers supported in SharePoint 2013](supported-mobile-device-browsers.md). 
   
-For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Mobile architecture in SharePoint 2013](https://zoom.it/E1BL). Also, for more information about how to administer your mobile environment, see [Administer mobile devices in SharePoint Server](/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
+For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Mobile architecture in SharePoint 2013](https://docs.microsoft.com/en-us/sharepoint/administration/mobile-devices-overview). Also, for more information about how to administer your mobile environment, see [Administer mobile devices in SharePoint Server](/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/mobile-devices-overview.md]:
```suggestion
For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Overview of mobile devices and SharePoint 2013](https://docs.microsoft.com/en-us/sharepoint/administration/mobile-devices-overview). Also, for more information about how to administer your mobile environment, see [Mobile administration (SharePoint Server 2010)](https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
```

File: SharePoint/SharePointServer/administration/mobile-devices-overview.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ SharePoint Server 2013 offers new, optimized viewing experiences across differen
 > [!IMPORTANT]
 > For more information about supported mobile browsers with SharePoint Server 2013, see [Mobile device browsers supported in SharePoint 2013](supported-mobile-device-browsers.md). 
   
-For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Mobile architecture in SharePoint 2013](https://zoom.it/E1BL). Also, for more information about how to administer your mobile environment, see [Administer mobile devices in SharePoint Server](/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
+For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Overview of mobile devices and SharePoint 2013](https://docs.microsoft.com/en-us/sharepoint/administration/mobile-devices-overview). Also, for more information about how to administer your mobile environment, see [Mobile administration (SharePoint Server 2010)](https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
```
Comment (nenonix) [inline SharePoint/SharePointServer/administration/mobile-devices-overview.md]:
```suggestion
For an end-to-end look and understanding of the SharePoint Server 2013 mobile landscape, see the poster [Overview of mobile devices and SharePoint 2013](https://docs.microsoft.com/sharepoint/administration/mobile-devices-overview). For more information about how to administer your mobile environment, see [Mobile administration (SharePoint Server 2010)](https://docs.microsoft.com/previous-versions/office/sharepoint-server-2010/ff393820(v=office.14)).
```

**Comment by scanum** [comment]
@tomresing

**Comment by scanum** [comment]
@tomresing Any updates in merging this PR? Changes look correct. Thanks

**Pull request merged**